src/mlpack/core/data/serialization_shim.hpp

```diff
@@ -123,13 +135,13 @@ inline FirstShim<T> CreateNVP(
 template<typename T>
 inline
 #ifndef BOOST_NO_FUNCTION_TEMPLATE_ORDERING
-const // Imitate the boost::serialization make_nvp() function.
+    const // Imitate the boost::serialization make_nvp() function.
 #endif
-boost::serialization::nvp<T> CreateNVP(
-    T& t,
-    const std::string& name,
-    typename std::enable_if_t<!HasSerialize<T>::value>* = 0,
-    typename std::enable_if_t<!std::is_pointer<T>::value>* = 0)
+    boost::serialization::nvp<T>
+    CreateNVP(T& t,
+              const std::string& name,
+              typename std::enable_if_t<!HasSerialize<T>::value>* = 0,
+              typename std::enable_if_t<!std::is_pointer<T>::value>* = 0)
```

Comment:
I think adding `clang-format` comments all over the code just for the sake of formatting is really confusing and we should definitely try to avoid that.  But I know that makes the clang-format configuration process harder... sorry about that... :(